traits/observation/_dsl_grammar.lark

```diff
@@ -3,52 +3,36 @@
 //     $ python etstool.py generate-parser
 //
 
-?start: expr
-
-// Connectors
-_dot: "."
-_colon: ":"
-
-// Grouped elements, e.g. "[a.b]", "[a,b]"
-?grouped: "[" parallel "]"
-
 // Simple name as trait name, e.g. "a"
-trait: NAME -> trait
```

Comment:
The "-> trait" part here is an alias, but it's aliasing "trait" to itself. I removed this and other self-aliasing cases, to make the grammar easier to parse mentally without needing to know about this particular lark feature.